.agents/skills/cuopt-server-api-python/assets/milp_basic/client.py

```
@@ -0,0 +1,82 @@
+
+import requests
+
+SERVER = os.environ.get("CUOPT_SERVER_URL", "http://localhost:8000")
+HEADERS = {"Content-Type": "application/json", "CLIENT-VERSION": "custom"}
+
+
+def server_ok():
+    try:
+        r = requests.get(f"{SERVER}/cuopt/health", timeout=2)
+        return r.status_code == 200
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)